.github/workflows/stale_marker.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Close stale issues and PR'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 45 days with no activity. Remove stale label or comment or this will be closed in 7 days.'
```

Comment:
too short period of time

Reply:
I usually look through the issues and manually close those ones that did not have activity during past 1.5-2 months. What is your suggestion about period? 3 months?

Reply:
yeah, let it be 3 month